simdutf/simdutf.cpp

```diff
@@ -6777,7 +6777,7 @@ internal::atomic_ptr<const implementation>& get_default_implementation() {
   return get_active_implementation();
 }
 #endif
-#define SIMDUTF_GET_CURRENT_IMPLEMENTION
+#define SIMDUTF_GET_CURRENT_IMPLEMENTATION
```

Comment:
I'm somewhat confused that this `#define` is changed only in one location. Is it unused anywhere else? 

We should probably update `simdutf` to the latest release anyway, we are quite far behind.